### DISCUSSION TITLE
2020 总结 | XX’s Blog

### DISCUSSION THREAD

**Discussion by xxchan** [xxchan/xxchan.github.io#21] [Announcements]
https://xxchan.github.io/life/2021/01/20/2020-review.html 

 Just some random thoughts of mine.

**Comment by izackwu** [comment]
作者的心态是真的优秀，值得学习

**Comment by vickiegpt** [comment]
太难了，sjtu申请比我们简单多了。

**Comment by godnoshadow** [comment]
或许你说的对，运气最重要，我之前一直为自己当初的选择后悔，看了这篇文章，我心态也开始平和一点了。
